packages/chart/src/ChartUtils.ts

```diff
@@ -757,13 +805,60 @@ class ChartUtils {
       seriesData.marker.color = shapeColor;
     }
 
+    if (shape != null && shape.length > 0) {
+      try {
+        seriesData.marker.symbol = ChartUtils.getMarkerSymbol(shape);
+      } catch (e) {
+        log.warn('Unable to handle shape', shape, ':', e);
+      }
+    }
+
+    if (shapeSize != null) {
+      seriesData.marker.size = shapeSize * ChartUtils.DEFAULT_MARKER_SIZE;
+    }
+
     // Skipping pie charts
     // Pie slice visibility is configured in chart layout instead of series data
     if (seriesVisibility != null && plotStyle !== dh.plot.SeriesPlotStyle.PIE) {
       seriesData.visible = seriesVisibility;
     }
   }
 
+  /**
+   * Get the marker symbol for the provided Deephaven shape
+   * Deephaven shapes: https://deephaven.io/enterprise/docs/plotting/visual-formatting/#point-formatting
+   * Plotly shapes: https://plotly.com/javascript/reference/scattergl/#scattergl-marker-symbol
+   * Table of plotly shapes: https://plotly.com/python/marker-style/
+   * @param shape Shape to get the marker symbol for
+   */
+  static getMarkerSymbol(shape: string): MarkerSymbol {
```

Comment:
Can you rename the param to `deephavenShape` just so it's a bit more clear which one is the param